### ISSUE TITLE
Express 3.x Support

### ISSUE THREAD

**Issue by jpillora** [panta/express-mongoose-resource#2]
app.resource is undefined and I presume it's because this was built for Express 2.x

**Comment by jpillora** [comment]
I (think I've) got it working by swapping `express.HTTPServer::resource` for  `express.application.resource` though maybe other parts need to be updated ?

**Comment by panta** [comment]
Hey, sorry for the huge delay!
You're right, express-mongoose-resource was built for express 2.x. Now that express-resource (which is an indirect dependency), has been updated to support express 3.x, I could try to update too, but I'd like to do so in a way to keep backward compatibility with 2.x.
You could also try https://github.com/panta/express-mongoose-crud (which also targets 2.x but should be way easier to change for 3.x. And I'll do that soon).

**Comment by jpillora** [comment]
Hey man, it's okay, I've rolled my own since I've got a bigger project in
mind, so I'm going to bake in user roles and other cool stuff :)

**Comment by simonplend** [comment]
@panta It would be really handy if at the top of the project README it mentioned that express-mongoose-resource was built for express 2.x as I've just wasted a bunch of time trying to get it working with the latest version of express!

**Comment by jpillora** [comment]
@simonplend [README.md](https://github.com/panta/express-mongoose-resource/blob/master/README.md) then click "Edit", save then pull request

**Comment by simonplend** [comment]
@jpillora Fair point! Done: https://github.com/panta/express-mongoose-resource/pull/5

**Comment by honeyd** [comment]
Any movement on modifying express-mongoose-resource to play nice with express 3.x?

**Comment by gigix** [comment]
Actually, any progress here? Or if all the authors are busy, please give some hints and I'll be happy to make it happen. I have a project with almost exact the requirement to expose mongoose resources, and I really don't want to handwrite all these boring CRUD every time.
